lib/matplotlib/axes/_axes.py

```diff
@@ -5168,9 +5176,20 @@ def _pcolorargs(funcname, *args, **kw):
         # is False.
 
         allmatch = kw.pop("allmatch", False)
+        norm = kw.pop("norm", None)
+        cmap = kw.pop("cmap", None)
+        alpha = kw.pop("alpha", 1)
 
         if len(args) == 1:
             C = np.asanyarray(args[0])
+            isBivari = (isinstance(norm, mcolors.BivariateNorm) or
```

Comment:
The previous change was reverted because two different variables `norm` and `cmap` were checked for their respective instances.